src/core/server/integration_tests/ci_checks/saved_objects/check_registered_types.test.ts

```diff
@@ -61,131 +61,131 @@ describe('checking migration metadata changes on all registered SO types', () =>
-        "osquery-pack": "784df8f1045e38a3fff4b77ce9aa729bfc0c3058",
-        "osquery-pack-asset": "42d5503cd17e1a08e7d822843934f3c92972e246",
-        "osquery-saved-query": "a8ef11610473e3d1b51a8fdacb2799d8a610818e",
+        "osquery-pack": "2845bec873d22246bc16832df9aca524972f74d4",
+        "osquery-pack-asset": "25d763f5a322070861407017fc8620dbab705778",
+        "osquery-saved-query": "a71e5e0a565ad2c9bdb4f01e4dfa200e1f46a754",
         "policy-settings-protection-updates-note": "c05c4c33a5e5bd1fa153991f300d040ac5d6f38d",
-        "privilege-monitoring-status": "4daec76df427409bcd64250f5c23f5ab86c8bac3",
+        "privilege-monitoring-status": "ada53885460c5ea6b80d46df79d7e68c507e60b4",
         "privmon-api-key": "c06b1614786ce7271087378b47d465c956ab1537",
-        "product-doc-install-status": "f94e3e5ad2cc933df918f2cd159044c626e01011",
-        "query": "1966ccce8e9853018111fb8a1dee500228731d9e",
-        "risk-engine-configuration": "f5ca37ab60d0bb0756869c9a4171146afbdd67a7",
-        "rules-settings": "53f94e5ce61f5e75d55ab8adbc1fb3d0937d2e0b",
+        "product-doc-install-status": "6c39628d11bfb493426c9db915f19f2e45699249",
+        "query": "d38c764b8faf414d0d2be53ec936d3d6ecd28f17",
+        "risk-engine-configuration": "726b14f68938267f3841f5026c55abb3205082b3",
+        "rules-settings": "a3570971f75ae2e8dd9d8b475e52b968cd1fa030",
         "sample-data-telemetry": "c38daf1a49ed24f2a4fb091e6e1e833fccf19935",
-        "search": "d81feb3845eb84c00dabfd3e89dc798c854c07dd",
+        "search": "f75fedbf08c1faf7a748cf18b1a834c55ba092b6",
         "search-session": "fae0dfc63274d6a3b90ca583802c48cab8760637",
         "search-telemetry": "1bbaf2db531b97fa04399440fa52d46e86d54dd8",
-        "search_playground": "3eba7e7c4563f03f76aea02f5dd3a7a739bf51a3",
+        "search_playground": "ec10848230e6b8c1471b771018fe6f4989ee6bd6",
         "security-ai-prompt": "1fc1c56cc078ed2c5506bb5a4e09f6876d02c97c",
         "security-rule": "151108f4906744a137ddc89f5988310c5b9ba8b6",
         "security-solution-signals-migration": "0be3bed0f2ff4fe460493751b8be610a785c5c98",
-        "security:reference-data": "3b68791ae5b2312593eee105277cdc3918f39ed7",
+        "security:reference-data": "182cfae072f836cc27b3e6c5b6f60ab88a489fc2",
         "siem-detection-engine-rule-actions": "123c130dc38120a470d8db9fed9a4cebd2046445",
-        "siem-ui-timeline": "9906092f527a21abdfab10e43c86b228ebc3861b",
+        "siem-ui-timeline": "c372bcb32251d2d7aab848a92ed7b26fcfdbe2d4",
         "siem-ui-timeline-note": "13c9d4c142f96624a93a623c6d7cba7e1ae9b5a6",
         "siem-ui-timeline-pinned-event": "96a43d59b9e2fc11f12255a0cb47ef0a3d83af4c",
-        "slo": "79878844eda6ac3d858c19255d9714752b1bb081",
+        "slo": "fd6b526eeb4f5b2e9d423ceb3abf51ee94867048",
         "slo-settings": "9a74f29769cb973a726852fcb1129481b45ce577",
-        "space": "758dd69293d1cd5a0190753cfd93101fe7693604",
+        "space": "c85816db41bf3b9e3b77194cfa6780e891508eee",
```

Comment:
We expect that all the hashes for the SO types that use modelVersions will change because `getMigrationHash` has changed and now includes `modelVersionHash` and the schemas' property hashes.